ports/zephyr/mpconfigport.h

```diff
@@ -72,18 +71,22 @@
 #define MICROPY_PY_MACHINE_PIN_MAKE_NEW mp_pin_make_new
 #define MICROPY_PY_MACHINE_UART     (1)
 #define MICROPY_PY_MACHINE_UART_INCLUDEFILE "ports/zephyr/machine_uart.c"
-#define MICROPY_PY_STRUCT           (0)
 #ifdef CONFIG_NETWORKING
 // If we have networking, we likely want errno comfort
 #define MICROPY_PY_ERRNO            (1)
 #define MICROPY_PY_SOCKET           (1)
 #endif
 #ifdef CONFIG_BT
 #define MICROPY_PY_BLUETOOTH        (1)
+#define MICROPY_PY_BUILTINS_SET     (1)
+#define MICROPY_PY_STRUCT           (1)
 #ifdef CONFIG_BT_CENTRAL
 #define MICROPY_PY_BLUETOOTH_ENABLE_CENTRAL_MODE (1)
 #endif
 #define MICROPY_PY_BLUETOOTH_ENABLE_GATT_CLIENT (0)
+#else
+#define MICROPY_PY_BUILTINS_SET     (0)
+#define MICROPY_PY_STRUCT           (0)
```

Comment:
I think it's best to just remove these settings from this config file completely and let them take their default values at all times (which means they'll be enabled for all boards, which is fine, they are core parts of Python).

Reply:
OK, will do.